baybe/utils/chemistry.py

```diff
@@ -26,8 +27,16 @@
 
 
 # Caching
-_cachedir = Path.home() / ".baybe_cache"
-_memory_utils = Memory(_cachedir / "utils")
+_cachedir = os.environ.get(
+    "BAYBE_CACHE_DIR", str(Path(tempfile.gettempdir()) / ".baybe_cache")
+)
+
+
+def _dummy_wrapper(func):
+    return func
+
+
+_disk_cache = _dummy_wrapper if _cachedir == "" else Memory(Path(_cachedir)).cache
```

Comment:
What happens here if the user defines an unreasonable/faulty `BAYBE_CACHE_DIR`? Will this then fail, crash, do nothing?

Reply:
it should fail within the `Memory` functionality when attempting to create that folder

Reply:
"It should" means "You expect this behavior" or "you have tested and verified this behavior"?

Reply:
i did not test but why would I test that, this function creates folders, if it gets a path it does what all functions do that try to create folders -> fail (= expected behavior)